datatypes/src/operations/image/colorizer.rs

```diff
@@ -669,6 +802,34 @@ mod tests {
         assert_eq!(color_table[2], RgbaColor::white());
     }
 
+    #[test]
+    fn logarithmic_color_table_with_extrema() {
+        let colorizer = Colorizer::logarithmic_gradient(
+            vec![
+                (5.0, RgbaColor::black()).try_into().unwrap(),
+                (15.0, RgbaColor::white()).try_into().unwrap(),
+            ],
+            RgbaColor::transparent(),
+            RgbaColor::pink(),
+            RgbaColor::pink(),
```

Comment:
Please use different colors here, so that you can test that you are not mistakenly switching colors, etc.

Reply:
https://github.com/geo-engine/geoengine/pull/684/commits/cd8b2dc6a2dfab354845303f76b731d2185fce6c